### PR TITLE
Add initial GitHub actions setup

### PR DESCRIPTION
**Pull request by dualcnhq**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 15.x
+    
+    - name: Install dependencies 
+      run: yarn install
+
+    - name: Format code
+      run: yarn run format
```
Comment (jbuckner) [inline .github/workflows/ci.yml]: I wouldn't run formatting during CI. We have a pre-commit git hook that will make sure files are formatted before being committed to the repo.

File: demo/index.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
   <script type="module">
     import { html, render } from 'lit-html';
-      import '../dist/index.js';
+    import '../dist/index';
```
Comment (jbuckner) [inline demo/index.html]: Does this work without the `.js`? I _thought_ you needed it in `html` files (even though you get that error saying it shouldn't be there). I think there's an eslint setting to ignore the extension in html files.

Comment (dualcnhq) [reply demo/index.html]: it still works without the `.js` extension there

File: .github/workflows/ci.yml (hunk 1 above)
Comment (dualcnhq) [reply .github/workflows/ci.yml]: updated and removed format code in CI

File: demo/index.html (hunk 2 above)
Comment (dualcnhq) [reply demo/index.html]: added eslint setting check instead

Comment (jbuckner) [reply demo/index.html]: Oh I was totally cool with this working without the `.js`! 😄 When I tried it, I wasn't able to get it working for some reason, but if it was working for you, let's drop the `.js` and you can get rid of the eslint exception.

**Review by jbuckner — APPROVED** [review]

**Pull request merged**